nxcompshad/src/Makefile.am

```diff
@@ -0,0 +1,45 @@
+NULL =
+
+lib_LTLIBRARIES = libXcompshad.la
+
+libXcompshad_la_SOURCES =				\
+    Core.cpp						\
+    Input.cpp						\
+    Logger.cpp						\
+    Manager.cpp						\
+    Shadow.cpp						\
+    Updater.cpp						\
+    Win.cpp						\
+    X11.cpp						\
+    $(NULL)
+
+libXcompshad_la_LIBADD =				\
+    @Xext_LIBS@						\
+    @Xdamage_LIBS@					\
+    @Xrandr_LIBS@					\
+    @Xtst_LIBS@						\
+    -L$(top_srcdir)/../nx-X11/exports/lib -lNX_X11	\
+    $(NULL)
+
+AM_CFLAGS =						\
+    $(Xext_CFLAGS)					\
+    $(Xdamage_CFLAGS)					\
+    $(Xrandr_CFLAGS)					\
+    $(Xtst_CFLAGS)					\
+    $(NULL)
+
+AM_CXXFLAGS =						\
+    $(BASE_CXXFLAGS)					\
+    $(NULL)
+
+AM_CPPFLAGS =						\
+    -I$(top_srcdir)/include				\
+    -I$(top_srcdir)/../nx-X11/exports/include		\
+    $(NULL)
+
+libXcompshad_la_LDFLAGS = -version-number @LT_COMPSHAD_VERSION@ -no-undefined
```

Comment:
ltmain.sh has "-version-number" as option and it is explicitly for situations like ours, where we don't obey the libtool devs' recommendations.

I will not change the .so name pattern here. This requires deeper investigation.